packages/itwinui-css/src/table/base.scss

```diff
@@ -66,7 +66,6 @@
       transform: translateX(50%);
       touch-action: none;
       cursor: ew-resize;
-      z-index: 1;
```

Comment:
We need this z-index to prevent the resizer getting clipped by the next column (which appears later in DOM order).

For sticky columns, you could add `z-index: 2` to make it appear on top of the resizers.

Reply:
Ohh, I see. Then I bumped the z-index of `.iui-table-sticky-cell` instead and confirmed that it works (https://github.com/iTwin/iTwinUI/pull/1943/commits/770fa884c8c9c98e5274a026bb10b4c55e275131).

Reply:
Can you also add a comment to explain the magic number?

Reply:
Sounds good, added (https://github.com/iTwin/iTwinUI/pull/1943/commits/5ce368724664efb76a181a754169b2d26777608e).

I also realized that it would be better to bump other z-indexes (paginator and the header wrapper) as a result of bumping the sticky column z-index. So, I made those changes too (https://github.com/iTwin/iTwinUI/pull/1943/commits/37891c36d3d770ca16c2139d1ec4e5584c7121dc).